tests/web/url.py

```diff
@@ -229,3 +265,4 @@ def test_parse_cp_1251(self):
         url = URL.from_url(url_string)
         self.assertEqual(url.host, 'test.com')
         self.assertEqual(url.query['query'], u'�'*6)
```

Comment:
Is it ok to write unprintable characters in code directly? Using `u'\N{REPLACEMENT CHARACTER}'` is more appropriate to me.